libs/orchestrator-ui-components/src/lib/components/StartPage/ListItemStartPage.tsx

```diff
@@ -1,9 +1,23 @@
 import React, { FC, useState } from 'react';
 import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiTextColor } from '@elastic/eui';
 import moment from 'moment';
-import { Process, Subscription } from '../../types';
+import { Process, ProductBase } from '../../types';
 import Link from 'next/link';
 
+interface Subscription {
+    name: string;
+    subscription_id: string;
+    description: string;
+    product: ProductBase;
+    product_id: string;
+    status: string;
+    insync: boolean;
+    customer_id: string;
+    start_date: number;
+    end_date: number;
+    note: string;
+}
+
 export interface ListItemStartPageProps {
```

Comment:
Out of scope: `ListItemStartPage` component needs to be covered by both flavours of UI integration test(s) namely Jest snapshot tests and Storybook stories towards avoiding unintentional regressions both in the rendered HTML and the UI itself respectively.

Reply:
Not now: when GraphQL also covers the data for these components